renovate.json

```diff
@@ -3,10 +3,43 @@
   "extends": [
     "config:recommended"
   ],
+  "regexManagers": [
```

Comment:
Do you see a warning about older format? Can we use `customManagers` of `"customType": "regex"`?

Reply:
Done changed `regexManagers` -> `customManagers` and added `"customType": "regex"`